### PR TITLE
remove unused annotation

### PR DESCRIPTION
**Pull request by zhyass**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

remove unused annotation

## Changelog

- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by PsiACE** [comment]
@zhyass Perhaps we can use `#[flaky_test]` to mark `meta_service::raftmeta_test::test_meta_node_snapshot_replication` as flaky test.

**Comment by zhyass** [comment]
> @zhyass Perhaps we can use `#[flaky_test]` to mark `meta_service::raftmeta_test::test_meta_node_snapshot_replication` as flaky test.

Sure, I will mark the test.

File: store/src/meta_service/raftmeta_test.rs [hunk 1]
```diff
@@ -381,6 +381,7 @@ async fn test_meta_node_add_database() -> anyhow::Result<()> {
 }
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 10)]
+#[flaky_test]
```
Comment (BohuTANG) [inline store/src/meta_service/raftmeta_test.rs:384]: Hi @zhyass ,
@drmingdrmer is working on it, so it's better to add it in another patch.

Comment (zhyass) [reply store/src/meta_service/raftmeta_test.rs:384]: ok

**Review by BohuTANG — APPROVED** [review]
LGTM
Thank you @zhyass

**Comment by BohuTANG** [comment]
The failed CI is not related to this pr, I merged.

**Pull request merged**